src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -190,29 +190,35 @@ int cloud_codec_encode_batch_data(
 void cloud_codec_populate_sensor_buffer(
 				struct cloud_data_sensors *sensor_buffer,
 				struct cloud_data_sensors *new_sensor_data,
-				int *head_sensor_buf);
+				int *head_sensor_buf,
+				size_t buffer_count);
```

Comment:
Is it intentional to include the buffer changes in the commit named `config: Make module independent and remove debug from prj.conf`?

Reply:
Yes. The commit moves the ```BUFFER_MAX``` configs from ```cloud_codec/Kconfig``` to ```Kconfig.data_module```. This was done to limit dependencies between Kconfigs in the source code. ```ringbuffer.c``` was using them to iterate over the buffers when it seemed cleaner to just pass in the size and only use the ```BUFFER_MAX``` configs int the data module when initially declaring the buffers.